Integrations/integration-EWSv2.yml

```diff
@@ -1162,6 +1166,12 @@ script:
             }
         }
 
+    def get_folder(folder_path, target_mailbox=None):
+        account = get_account(target_mailbox or ACCOUNT_EMAIL)
+        folder = get_folder_by_path(account, folder_path)
+        folderDict = parse_item_as_dict(folder, target_mailbox)
+
+        return get_entry_for_object("Folder %s" % (folder_path,), CONTEXT_UPDATE_FOLDER, folderDict)
```

Comment:
The context output is inconsistent with the context output of other folder-related commands.
For example, ews-find-folders.

This is the current implementation, with an example output of ews-find-folders as well:
![playground - war room 2018-08-26 14-10-58](https://user-images.githubusercontent.com/39116813/44627646-f2690900-a939-11e8-86b7-9f1a1aa8e2f6.jpg)

This is the expected context output of the ews-get-folder command:
![playground - war room 2018-08-26 14-14-19](https://user-images.githubusercontent.com/39116813/44627681-77542280-a93a-11e8-9d2f-98d07224d7e1.jpg)